dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/resource/ImageFileResourceRepositoryImpl.java

```diff
@@ -0,0 +1,134 @@
+package de.digitalcollections.cudami.server.backend.impl.jdbi.identifiable.resource;
+
+import de.digitalcollections.cudami.server.backend.api.repository.identifiable.IdentifierRepository;
+import de.digitalcollections.cudami.server.backend.api.repository.identifiable.resource.ImageFileResourceRepository;
+import de.digitalcollections.cudami.server.backend.impl.jdbi.identifiable.IdentifiableRepositoryImpl;
+import de.digitalcollections.model.api.identifiable.resource.ImageFileResource;
+import de.digitalcollections.model.api.paging.SearchPageRequest;
+import de.digitalcollections.model.api.paging.SearchPageResponse;
+import de.digitalcollections.model.impl.identifiable.parts.LocalizedTextImpl;
+import de.digitalcollections.model.impl.identifiable.resource.ImageFileResourceImpl;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import org.jdbi.v3.core.Jdbi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ImageFileResourceRepositoryImpl extends IdentifiableRepositoryImpl<ImageFileResource>
+    implements ImageFileResourceRepository {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(ImageFileResourceRepositoryImpl.class);
+
+  public static final String MAPPING_PREFIX = "fr";
+  public static final String TABLE_ALIAS = "f";
+  public static final String TABLE_NAME = "fileresources_image";
+
+  public static String getSqlInsertFields() {
+    return FileResourceMetadataRepositoryImpl.getSqlInsertFields() + ", height, width";
+  }
+
+  /* Do not change order! Must match order in getSqlInsertFields!!! */
+  public static String getSqlInsertValues() {
+    return FileResourceMetadataRepositoryImpl.getSqlInsertValues() + ", :height, :width";
+  }
+
+  public static String getSqlSelectAllFields(String tableAlias, String mappingPrefix) {
+    return getSqlSelectReducedFields(tableAlias, mappingPrefix)
+        + ", "
+        + tableAlias
+        + ".height "
+        + mappingPrefix
+        + "_height, "
+        + tableAlias
+        + ".width "
+        + mappingPrefix
+        + "_width";
+  }
+
+  public static String getSqlSelectReducedFields(String tableAlias, String mappingPrefix) {
+    return FileResourceMetadataRepositoryImpl.getSqlSelectReducedFields(tableAlias, mappingPrefix);
+  }
+
+  public static String getSqlUpdateFieldValues() {
+    return FileResourceMetadataRepositoryImpl.getSqlUpdateFieldValues()
+        + ", height=:height, width=:width";
+  }
+
+  private final FileResourceMetadataRepositoryImpl fileResourceMetadataRepositoryImpl;
```

Comment:
Vielleicht besser
```suggestion
  private final FileResourceMetadataRepositoryImpl metadataRepository;
```
?

Reply:
ok